[CI] IndexStats.testBulkStats fails looking for a stat that is zero

The failure looks like:

```
java.lang.AssertionError: 
Expected: a value greater than <0L>
     but: <0L> was equal to <0L>
	at __randomizedtesting.SeedInfo.seed([53E1A42B3AEEE880:D9991F5953912569]:0)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.elasticsearch.indices.stats.IndexStatsIT.testBulkStats(IndexStatsIT.java:1101)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
```

It doesn't fail with:

```
./gradlew ':server:integTest' --tests "org.elasticsearch.indices.stats.IndexStatsIT.testBulkStats" -Dtests.seed=53E1A42B3AEEE880 -Dtests.security.manager=true -Dtests.locale=id -Dtests.timezone=Australia/Brisbane -Dcompiler.java=14
```

But I have seen it a few times, @probakowski I believe this is related to #52208

--- a/server/src/internalClusterTest/java/org/elasticsearch/indices/stats/IndexStatsIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/indices/stats/IndexStatsIT.java
@@ -85,6 +85,7 @@ import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertSear
 import static org.hamcrest.Matchers.emptyCollectionOf;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.lessThan;
 import static org.hamcrest.Matchers.notNullValue;
@@ -1066,7 +1067,6 @@ public class IndexStatsIT extends ESIntegTestCase {
         assertThat(response.getTotal().queryCache.getMemorySizeInBytes(), equalTo(0L));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/55485")
     public void testBulkStats() throws Exception {
         final String index = "test";
         assertAcked(prepareCreate(index).setSettings(settingsBuilder().put("index.number_of_shards", 2)
@@ -1086,15 +1086,15 @@ public class IndexStatsIT extends ESIntegTestCase {
         IndicesStatsResponse stats = client().admin().indices().prepareStats(index).setBulk(true).get();
 
         assertThat(stats.getTotal().bulk.getTotalOperations(), equalTo(4L));
-        assertThat(stats.getTotal().bulk.getTotalTimeInMillis(), greaterThan(0L));
+        assertThat(stats.getTotal().bulk.getTotalTimeInMillis(), greaterThanOrEqualTo(0L));
         assertThat(stats.getTotal().bulk.getTotalSizeInBytes(), greaterThan(0L));
-        assertThat(stats.getTotal().bulk.getAvgTimeInMillis(), greaterThan(0L));
+        assertThat(stats.getTotal().bulk.getAvgTimeInMillis(), greaterThanOrEqualTo(0L));
         assertThat(stats.getTotal().bulk.getAvgSizeInBytes(), greaterThan(0L));
 
         assertThat(stats.getPrimaries().bulk.getTotalOperations(), equalTo(2L));
-        assertThat(stats.getPrimaries().bulk.getTotalTimeInMillis(), greaterThan(0L));
+        assertThat(stats.getPrimaries().bulk.getTotalTimeInMillis(), greaterThanOrEqualTo(0L));
         assertThat(stats.getPrimaries().bulk.getTotalSizeInBytes(), greaterThan(0L));
-        assertThat(stats.getPrimaries().bulk.getAvgTimeInMillis(), greaterThan(0L));
+        assertThat(stats.getPrimaries().bulk.getAvgTimeInMillis(), greaterThanOrEqualTo(0L));
         assertThat(stats.getPrimaries().bulk.getAvgSizeInBytes(), greaterThan(0L));
     }
 

